src/student_success_tool/analysis/pdp/dataops.py

```diff
@@ -198,6 +161,84 @@ def standardize_course_dataset(df: pd.DataFrame) -> pd.DataFrame:
     )
 
 
+def clean_up_labeled_dataset_cols_and_vals(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Drop a bunch of columns not needed or wanted for modeling, and set to null
+    any values corresponding to time after a student's current year of enrollment.
+    """
+    return (
+        df.pipe(
+            drop_columns_safely,
+            cols=[
+                # metadata
+                "institution_id",
+                "term_id",
+                "academic_year",
+                "academic_term",
+                "cohort",
+                "cohort_term",
```

Comment:
Hypothesis: students that first enroll in the Fall are different than students that first enroll in the Spring/Summer

Reply:
Same here.